warning: `int QWheelEvent::x() const` is deprecated

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/PlotCanvas.cpp: In member function ‘virtual void OpenMS::PlotCanvas::wheelEvent(QWheelEvent*)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/PlotCanvas.cpp:181:15: warning: ‘int QWheelEvent::x() const’ is deprecated: Use position() [-Wdeprecated-declarations]
  181 |     zoom_(e->x(), e->y(), e->delta() > 0);
      |           ~~~~^~
In file included from /usr/include/qt/QtGui/qtextlayout.h:49,
                 from /usr/include/qt/QtGui/qabstracttextdocumentlayout.h:45,
                 from /usr/include/qt/QtGui/QtGui:5,
                 from /usr/include/qt/QtWidgets/QtWidgetsDepends:4,
                 from /usr/include/qt/QtWidgets/QtWidgets:3,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/AxisWidget.h:41,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/PlotCanvas.cpp:38:
/usr/include/qt/QtGui/qevent.h:229:16: note: declared here
  229 |     inline int x() const { return int(p.x()); }
      |                ^
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/PlotCanvas.cpp:181:23: warning: ‘int QWheelEvent::y() const’ is deprecated: Use position() [-Wdeprecated-declarations]
  181 |     zoom_(e->x(), e->y(), e->delta() > 0);
      |                   ~~~~^~
In file included from /usr/include/qt/QtGui/qtextlayout.h:49,
                 from /usr/include/qt/QtGui/qabstracttextdocumentlayout.h:45,
                 from /usr/include/qt/QtGui/QtGui:5,
                 from /usr/include/qt/QtWidgets/QtWidgetsDepends:4,
                 from /usr/include/qt/QtWidgets/QtWidgets:3,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/AxisWidget.h:41,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/PlotCanvas.cpp:38:
/usr/include/qt/QtGui/qevent.h:231:16: note: declared here
  231 |     inline int y() const { return int(p.y()); }
      |                ^

```

--- a/src/openms_gui/source/VISUAL/PlotCanvas.cpp
+++ b/src/openms_gui/source/VISUAL/PlotCanvas.cpp
@@ -178,7 +178,13 @@ namespace OpenMS
 
   void PlotCanvas::wheelEvent(QWheelEvent* e)
   {
+     /* Supressed warning int QWheelEvent::x() const and y() deprecated
+     * Use position() instead, from Qt 5.14
+     */
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
     zoom_(e->x(), e->y(), e->delta() > 0);
+#pragma GCC diagnostic pop
     e->accept();
   }
 

